src/components/Gradebook/index.jsx

```diff
@@ -433,13 +443,40 @@ export default class Gradebook extends React.Component {
                   open={this.state.modalOpen}
                   title="Edit Grades"
                   closeText="Cancel"
-                  body={(
+                  body={( // UI for this component
                     <div>
-                      <h3>{this.state.modalModel[0].assignmentName}</h3>
+                      <div>
+                        <div style={{ float: 'left', paddingRight: '49px' }}>Assignment: </div> <div>{this.state.modalAssignmentName}</div>
```

Comment:
Please get rid of the style tag and replace with either some class names or some of bootstraps utility classes. https://getbootstrap.com/docs/4.1/layout/utilities-for-layout/